src/Simplex/Chat/Store/Direct.hs

```diff
@@ -253,6 +254,12 @@ getDeletedContacts db user@User {userId} = do
 getDeletedContact :: DB.Connection -> User -> Int64 -> ExceptT StoreError IO Contact
 getDeletedContact db user contactId = getContact_ db user contactId True
 
+-- deleteContactProfile_ :: DB.Connection -> UserId -> Contact -> IO ()
+-- deleteContactProfile_ db userId Contact {contactId} = do
+--   -- TODO not quite clear why there is more than one profile in this case
```

Comment:
Don't understand this comment.. and how a contact_id can have multiple profiles. But this is new code, isn't it?